makefile

```diff
@@ -25,9 +25,9 @@ build :
 get :
 	@echo "download 3rd party packages...."
 	@$(GO_GET) github.com/gorilla/mux
-	@$(GO_GET) github.com/spf13/cobra
 	@$(GO_GET) github.com/ghodss/yaml
 	@$(GO_GET) github.com/andygrunwald/go-jira
+	@$(GO_GET) github.com/trivago/tgo/tcontainer
```

Comment:
Why do we need this import?

Reply:
@jerbia this package is used in alertmgr/jira.go (line 616).